nanoqm/workflows/input_validation.py

```diff
@@ -256,6 +262,22 @@ def add_multiplicity(self) -> None:
                 s.specific.cp2k.force_eval.dft.multiplicity = self.general['multiplicity']
                 s.specific.cp2k.force_eval.dft.uks = ""
 
+    def add_functional_x(self) -> None:
+        """Add the keyword for the exchange part of the DFT functional: GGA or MGGA"""
+        for s in (
```

Comment:
```suggestion
        self.general['functional_x'] is None:
            return
        for s in (
```
Skip the rest of the method if `functional_x` has not been specified.